tdrs-backend/tdpservice/settings/common.py

```diff
@@ -210,18 +210,28 @@ class Common(Configuration):
                 "class": "logging.StreamHandler",
                 "formatter": "simple",
             },
+            "application": {
+                "class": "logging.StreamHandler",
+                "formatter": "verbose",
+                "stream": sys.stdout,
+            },
         },
         "loggers": {
+            "tdpservice": {
+               "handlers": ["application"],
+               "propagate": True,
+               "level": LOGGING_LEVEL
+            },
             "django": {"handlers": ["console"], "propagate": True},
             "django.server": {
                 "handlers": ["django.server"],
-                "level": LOGGING_LEVEL,
                 "propagate": False,
+                "level": LOGGING_LEVEL
```

Comment:
I misunderstood that levels must be set in the loggers not handlers. When I added these back in, I made them visually consistent with our new app-level logger and put `level` as the last param.